lib/steps.js

```diff
@@ -92,7 +92,7 @@ BeforeAll(async () => {
   // Make the folder to store all the files for this run of the tests
   scope.paths.artifacts = files.make_artifacts_folder();
   // Save its name in a file. `session_vars` can get it back later.
-  // This is used by at least clean_reports.js.
+  // This is used by at least run_cucumber.js.
```

Comment:
Did I miss this? I don't see `save_artifacts_path_name` in run_cucumber.js. Do we need to put it somewhere? Where is the artifacts path name being saved right now? steps.js? Something in the setup?

Reply:
```suggestion
  // The value we are saving is used by at least run_cucumber.js.
```